econml/ortho_forest.py

```diff
@@ -855,3 +908,113 @@ def _check_treatment(self, T):
         except Exception as exc:
             raise ValueError("Expected numeric array but got non-numeric types.")
         return T
+
+
+class BLBInference(Inference):
+    """
+    Bootstrap-of-Little-Bags inference implementation for the OrthoForest classes.
+
+    This class can only be used for inference with any estimator derived from :class:`BaseOrthoForest`.
+
+    Parameters
+    ----------
+    estimator : :class:`BaseOrthoForest`
+        Estimator to perform inference on. Must be a child class of :class:`BaseOrthoForest`.
+    """
+
+    def fit(self, estimator, *args, **kwargs):
```

Comment:
Should you return self at the end?

Reply:
(I can't remember whether we care about this at all for inference objects)

Reply:
I think's it's good practice to return `self`. Fixed!